rethinkdb/rethinkdb.d.ts

```diff
@@ -49,19 +79,61 @@ declare module "rethinkdb" {
         close(): Promise<void>;
     }
 
+    interface Row extends Expression<any> {
+      (name: string): Expression<any>;
+    }
+
+    /**
+     * Connection options.
+     *
+     * See: https://rethinkdb.com/api/javascript/connect/
+     */
     interface ConnectionOptions {
-        host: string;
-        port: number;
+        /** The host to connect to (default `localhost`) */
+        host?: string;
+
+        /** The port to connect on (default `28015`) */
+        port?: number;
+
+        /** The default database (default `test`) */
         db?: string;
-        authKey?: string;
```

Comment:
I see `authKey` in one of the examples inhttps://rethinkdb.com/api/javascript/connect/. why has it been removed?